manifests/deploy/openshift/metering-operator/metering-operator-deployment.yaml

```diff
@@ -54,12 +54,12 @@ spec:
         - name: RELEASE_HISTORY_LIMIT
           value: "3"
         resources:
-          limits:
-            cpu: 50m
-            memory: 100Mi
-          requests:
-            cpu: 50m
-            memory: 100Mi
+            limits:
+              cpu: 50m
+              memory: 100Mi
+            requests:
+              cpu: 50m
+              memory: 100Mi
 
```

Comment:
Is everything indented 1 too far in from 57-62?

Reply:
Yeah, that looks too far.